app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TaskDaoTest.kt

```diff
@@ -44,14 +44,9 @@ class TaskDaoTest {
     ).allowMainThreadQueries().build()
 
     // Set the main coroutines dispatcher for unit testing.
-    @ExperimentalCoroutinesApi
```

Comment:
Note this was redundant since `@ExperimentalCoroutinesApi` is added before the class declaration